src/Lykke.Service.B2c2Adapter/Services/OrderBooksService.cs

```diff
@@ -192,9 +272,38 @@ private static IEnumerable<OrderBookItem> GetOrderBookItems(IEnumerable<Quantity
             return result;
         }
 
+        private void SetHelthCheck(PriceMessage result)
+        {
+            if (_healthCheck.TryGetValue(result.Instrument, out var counter))
+                _healthCheck[result.Instrument] = counter + 1;
+            else
+                _healthCheck[result.Instrument] = 1;
+        }
+
+        private Task WriteHealthCheck()
+        {
+            var list = _healthCheck.OrderBy(x => x.Value)
+                                   .Select(x => x.Key)
+                                   .Select(key => $"{key} : {_healthCheck[key]}")
+                                   .ToList();
+            try
+            {
+                _log.Info($"Health check: {Environment.NewLine}{string.Join(Environment.NewLine, list)}");
+            }
+            catch (Exception ex)
```

Comment:
logger does not throw exceptions, why catch?

Reply:
forget to remove after rewrite, fixed